packages/figma/src/lib/TokensData.ts

```diff
@@ -0,0 +1,69 @@
+export type TokenPrimitiveValue = string | number
+
+export type TokenValue = {
+  value: TokenPrimitiveValue
+  description?: string
+}
+
+export type TokensGroup = { [key: string]: TokenPrimitiveValue | TokenValue | TokensGroup }
+
+export const isTokenPrimitiveValue = (value: any): value is TokenPrimitiveValue => {
```

Comment:
В тайп-гардах принятно объявлять "неизвестный" тип:

```suggestion
export const isTokenPrimitiveValue = (value: unknown): value is TokenPrimitiveValue => {
```